### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in 996 end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7316]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the `/disability-benefits/996` app folder, I wasn't sure if there were modals or hidden content that needed axe checks. I'd like the front-end engineering team to review this application, and add end-to-end tests as needed. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications/disability-benefits/996/tests`
* `$ yarn test:e2e src/applications/disability-benefits/996/tests`

## Acceptance Criteria

**Definition of done:**
- [ ] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [ ] axe checks are run for hidden content like modal windows, accordions
- [ ] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [ ] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by 1Copenut** [comment]
@luke-gcio If the team has moved to use the Cypress forms testing, would you mind commenting here and at include me? The Cypress forms tester has axe checks built in and would be a good way to close this issue without additional work.

**Comment by 1Copenut** [comment]
@luke-gcio Bumping this for visibility. Thank you!

**Comment by luke-gcio** [comment]
Cypress tests are enabled for HLR.
